codeforboston/maple
# Newsfeed Testimony Card

# Summary

Issue #1633

# Checklist


# Screenshots

![image](https://github.com/user-attachments/assets/f669a5d4-3893-4cce-bff7-0c0d05060cce)

![image](https://github.com/user-attachments/assets/d8cee686-504a-482c-8a4a-f73e34f50354)

![image](https://github.com/user-attachments/assets/2f910ef4-5878-4951-a26a-9485692da037)

# Known issues

Future Additions:
1. pagination
2. testimony id for View Full Testimony link
3. testimony card has testimony text but also needs bill text
4. author type (is author an org or not?): so we can use seperate avatars for orgs and users
5. author id in order to link back to testimony author's profile page
  a. handle when going to a user profile page that is private
6. check: is isBillMatch currently using demo data?

# Steps to test/reproduce

1. Go to the newsfeed page
2. check desktop and mobile screen sizing
3. uncheck both filter boxes and check the "no results" card
